src/main/java/apoc/periodic/Periodic.java

```diff
@@ -163,6 +163,13 @@ private long executeNumericResultStatement(@Name("statement") String statement,
         return Stream.of(info);
     }
 
+    @Procedure(mode = Mode.WRITE)
+    @Description("apoc.periodic.repeat2('name',statement,repeat-rate-in-seconds, config) submit a repeatedly-called background statement with configuration")
```

Comment:
you don't need a second procedure

just add config to the regular one and default it to an empty map

Reply:
@jexp Okay will do.

Reply:
If we only use one procedure wouldn't it result in a breaking change?

Reply:
@jexp If we did that wouldn't it cause a breaking change? We would be going from 3 parameters to 4 parameters, even if we set the default value to an empty map.